File: DTOs/Sources/RideServiceDTOs/CreateRideDTO.swift
import Foundation

public struct CreateRideDTO: Codable {
    public var name: String
    public var description: String?
    public var starts: Date
    public var latitude: Float
    public var longitude: Float
    
    public init(name: String, description: String? = nil, starts: Date, latitude: Float, longitude: Float) {
        self.name = name
        self.description = description
        self.starts = starts
        self.latitude = latitude
        self.longitude = longitude
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/RideServiceDTOs/CreateRideDTO.kt
package net.ipv64.kivop.dtos.RideServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class CreateRideDTO (
    var name : String,
    var description : String?,
    var starts : LocalDateTime,
    var latitude : Float,
    var longitude : Float,
)
